roles/keycloak/tasks/main.yml

```diff
@@ -17,7 +17,7 @@
   vars:
     packages:
     - firewalld
-    - java-openjdk
+    - java-openjdk-headless
```

Comment:
I had to modify the OpenJDK dependency to specifically use 1.8.0 in my CI patch, so I just made this change as a part of that PR.  We can either drop this patch, or I can just rebase my work on top of this if we choose to merge your work first.